src/utils/components/PodSelectorModal/PodSelectorModal.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { FC, useState } from 'react';
+import * as _ from 'lodash';
```

Comment:
please remove

Reply:
why resolved?

Reply:
i removed lodash usage but forgot the import